website/docs/self-hosted-configuration.md

```diff
@@ -97,7 +97,7 @@ By default, Renovate will use the most efficient approach to updating package fi
 
 ## trustLevel
 
-Setting trustLevel to "high" can make sense in many self-hosted cases where the bot operator trusts the content in each repository.
+Setting trustLevel to `high` can make sense in many self-hosted cases where the bot operator trusts the content in each repository.
```

Comment:
For ones like this, I think it would be best to have them still with double quotes but then surrounded by backticks. e.g. like you did in some cases already: 
![image](https://user-images.githubusercontent.com/6311784/56089922-a653e000-5e9a-11e9-8a2c-6e4ff3d6d0e7.png)